Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec bin/rails s
+job: bundle exec sidekiq
```

Comment:
IIRC `foreman start -f .proc_file` would do it. No reason you can't keep your existing workflow and ignore this either - I've just found having a defined Procfile helpful in the past and I got a bit sad when I ran `foreman start` and nothing happened 😄